develop/tutorials/articles/110-portlets/02-liferay-soy-portlet/01-creating-a-soy-portlet.markdown

```diff
@@ -44,7 +44,7 @@ the OSGi metadata next.
 Add the OSGi metadata to your module's `bnd.bnd` file. A sample BND 
 configuration is shown below:
 
-    Bundle-Name: Liferay Hello Soy Web
+    Bundle-Name: hello-soy-web
```

Comment:
Correct, that was my initial thought.

Good to know there is no restriction on names and versions.